src/pages/request-term/index.tsx

```diff
@@ -0,0 +1,159 @@
+                color="#FEF9F9"
+                fontStyle="normal"
+                fontWeight="400"
+                title="request-a-term">
+                {t("requestATermTitle")}
+              </Heading>
+              <Text
+                title="request-a-term-description"
+                paddingTop="2"
+                width="332px"
+                height="25px"
+                fontFamily="Inter"
+                fontStyle="normal"
+                fontWeight="400"
+                fontSize="14px"
+                lineHeight="17px"
+                color="#FEF9F9"
+              >
+                {t("requestATermText")}
+              </Text>
+            </Box>
+
+            <Stack spacing={1}
+              paddingLeft={{ base: 10 }}
+              paddingRight={{ base: 10 }}>
+              <Center>
+                <form>
+                  <Flex
+                    direction="column"
+                    justifyContent="center"
+                    alignItems="start">
+                    <FormLabel
+                      color="#FFFDFD"
+                      htmlFor="Term"
+                      data-testid="term-label"
+                    >
+                      {t("termLabel")}
+                    </FormLabel>
+                    <Input
+                      data-testid="term-input"
+                      border="1px solid #000000"
+                      boxShadow="0px 4px 4px rgba(0, 0, 0, 0.25)"
+                      id="Term"
+                      backgroundColor="#FFFDFD"
+                      borderRadius="3px"
+                      height="37px"
+                      width={{ base: "300px", md: "526px", lg: "600px" }}
+                      marginTop={{base: 1}}
+                      marginBottom={{ base: 8, md: 6, lg: 4 }} />
+                    <FormLabel
+                      color="#FFFDFD"
+                      htmlFor="Category"
+                      data-testid="category-label">
+                      {t("termCategoryLabel")}
+                    </FormLabel>
+                    <Select
+                      data-testid="term-category-select"
+                      boxShadow="0px 4px 4px rgba(0, 0, 0, 0.25)"
+                      id="Category"
+                      placeholder="Select a Category"
+                      backgroundColor="#FFFDFD"
+                      borderRadius="3px"
+                      height="37px"
+                      width={{ base: "300px", md: "526px", lg: "600px" }}
+                      marginTop="1"
+                      marginBottom={{ base: 8, md: 6, lg: 4 }}>
+                      <option value="general">General</option>
+                      <option value="protocol">Protocol</option>
+                      <option value="dao">DAO</option>
+                      <option value="Token">Token</option>
+                      <option value="application">Application</option>
+                      <option value="finance">Finance</option>
```

Comment:
i recommend adding these in the locale file as well

Reply:
Awesome. I will include those, that totally slipped my mind